StratisCore.UI/src/app/wallet/smart-contracts/components/modals/transaction/transaction.component.ts

```diff
@@ -141,9 +140,9 @@ export class TransactionComponent implements OnInit {
 
         let gasLimitValidator = (this.mode === Mode.Call ? gasCallLimitMinimumValidator : gasCreateLimitMinimumValidator);
 
-        this.amount = new FormControl(0, [amountValidator]);
+        this.amount = new FormControl(0.1, [amountValidator]);
```

Comment:
Why is this set to 0.1 per default? Not too familiar with the smart contracts so I'm just wondering.

Reply:
Maciej comment was: `Contract deployment or call with amount 0 will not execute`, so basically we are just putting a meaningful default